client/src/locales/translations/es_ES.json

```diff
@@ -0,0 +1,766 @@
+        "Company": "Empresa",
+        "Community": "Comunidad",
+        "Government": "Gobierno",
+        "Non-profit": "Sin Fines de Lucro",
+        "University": "Universidad",
+        "Health care provider": "Proveedor de Servicios de Salud",
+        "Other": "Otros"
+      },
+      "typesOptions": {
+        "Medical Supplies": "Suministros Médicos",
+        "Groceries/Food": "Abarrotes/Alimentos",
+        "Business": "Negocios",
+        "Education": "Educación",
+        "Legal": "Legal",
+        "Wellbeing/Mental": "Bienestar/Mental",
+        "Entertainment": "Entretenimiento",
+        "Information": "Información",
+        "Funding": "Financiación",
+        "R&D": "I&D (Investigación y Desarrollo)",
+        "Tech": "Tecnología",
+        "Other": "Otros"
+      },
+      "lookingForOptions": {
+        "Request Help": "Solicitar Ayuda",
+        "Offer Help": "Ofrecer Ayuda"
+      }
+    },
+    "filterBy": "Filtrar por",
+    "noResults": "Lo sentimos, actualmente no hay publicaciones relevantes disponibles. Intenta utilizar un filtro de búsqueda diferente o <0> crea una publicación </0>"
+  },
+  "post": {
+    "create": "Crear una publicación",
+    "continueAs": "Continuar publicando como",
+    "individual": "Individuo",
+    "organisation": "Organización",
+    "givingHelp": "Ofreciendo ayuda",
+    "gettingHelp": "Solicitud de ayuda",
+    "writePostHere": "Escribe tu publicación aquí",
+    "title": "Título",
+    "writePost": "Escribe tu publicación",
+    "whatGiving": "¿Qué estás ofreciendo?",
+    "addTags": "Agrega etiquetas para que tu publicación sea más visible",
+    "whatVisibility": "¿Cuál es la visibilidad de tu publicación?",
+    "postVisible": "La publicación será visible",
+    "success": "Éxito",
+    "viewPost": "Ver tu Publicación",
+    "like": "Me gusta",
+    "like_plural": "Me gusta",
+    "likeWithCount": "{{count}} Me gusta",
+    "likeWithCount_plural": "{{count}} Me gusta",
+    "comment": "Comentar",
+    "comment_plural": "Comentarios",
+    "commentWithCount": "{{count}} Comentar",
+    "commentWithCount_plural": "{{count}} Comentarios",
+    "options": {
+      "shareWith": {
+        "city": "En mi Ciudad",
+        "state": "En mi Estado",
+        "worldwide": "Mundial",
+        "country": "En mi País"
+      },
+      "expires": {
+        "forever": "Para siempre",
+        "month": "Por un mes",
+        "week": "Por una semana",
+        "day": "Por un día"
+      },
+      "helpTypes": {
+        "request": "Busco ayuda",
+        "offer": "Ofrezco ayuda"
+      }
+    },
+    "post": "Publicación",
+    "postAsOrg": "Publicación como Organización",
+    "createNew": "Crear uno nuevo",
+    "whatNeed": "¿Qué necesitas?",
+    "viewLess": "Ver Menos",
+    "viewMore": "Ver Más",
+    "deletePostConfirmation": "¿Estás seguro de que deseas eliminar la publicación?",
+    "deleteCommentConfirmation": "¿Estás seguro de que quieres borrar el comentario?",
```

Comment:
You should change borrar everywhere to eliminar instead, or just change eliminar to borrar instead.